src/main.cpp

```diff
@@ -37,9 +37,9 @@ int main(const int argc, const char *const *const argv) {
     args::HelpFlag help(parser, "help", "Display this help text", {'h', "help"});
 
     args::ValueFlag<fs::path> appDirPath(parser, "appdir path", "Path to an existing AppDir", {"appdir"});
-    args::ValueFlagList<std::string> extraPlugins(parser, "plugin",
-                                                  "Extra Qt plugin to deploy (specified by name, filename or path)",
-                                                  {'p', "extra-plugin"});
+    args::ValueFlagList<std::string> extraModules(parser, "module",
+                                                  "Extra Qt module to deploy (specified by name, filename or path)",
+                                                  {'m', "extra-module"});
```

Comment:
While there's backwards compatibility for the environment variable `EXTRA_QT_PLUGINS`, the same isn't true for the CLI. Not necessarily problematic, but I hope the tool will at least report an error when somebody tries to use the old parameter.

Reply:
When one tries to use the old parameter it will show the help text, is that enough?

Reply:
That's helpful, and in addition the tool should return an error code so that people's CI builds would not report success regardless.